### PR TITLE
Add additional interfaces to Slurm login nodes

### PR DESCRIPTION
**Pull request by wiktorn**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

**Comment by mr0re1** [comment]
Thank you @wiktorn, change does LGTM, I'm running tests and will land it if it passes.

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf [hunk 1]
```diff
@@ -104,6 +104,19 @@ variable "login_nodes" {
       auto_delete  = optional(bool, true)
       boot         = optional(bool, false)
     })), [])
+    additional_networks = optional(list(object({
```
Comment (mr0re1) [inline community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf:107]: Could you please match the definition with one from `community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf : L404`. Those extra fields will remain inert (e.g. `queue_count`), but we would prefer to "over-provision". Thank you.

Comment (wiktorn) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf:107]: Thanks for the pointer. Added.

**Review by mr0re1 — APPROVED** [review]

**Comment by wiktorn** [comment]
Looks like the test job got stucked. Meanwhile, while testing I found out that template is correct, but instances are created with single interface, and this is probably due to explicitly specyfing `network_interfaces` in `_slurm_instance` [here](https://github.com/GoogleCloudPlatform/slurm-gcp/blob/master/terraform/slurm_cluster/modules/_slurm_instance/main.tf#L77), so it overrides whatever was provided in the template.

Let me check if this can be fixed here, or requires changes in `slurm-gcp`

**Comment by mr0re1** [comment]
@wiktorn  , I've just noticed that your PR was send against `main`. I've rebased it on `develop`, now there are some merge conflicts, please resolve. Everything else LGTM.

**Comment by wiktorn** [comment]
@mr0re1 rebased. I see that we have already `num_instances` to use, I'll create another PR for slurm-gcp for the other issue I discovered here.

**Review by mr0re1 — APPROVED** [review]

**Pull request merged**